lib/vet360/models/base.rb

```diff
@@ -11,6 +11,26 @@ class Base
       include Virtus.model(nullify_blank: true)
 
       SOURCE_SYSTEM = 'VETSGOV'
+      DEFAULT_ATTRS = %i[effective_start_date source_date vet360_id].freeze
+
+      def self.with_defaults(user, params)
+        instance = new(params)
+        now      = Time.now.iso8601
```

Comment:
We should probably use `Time.zone.now` because it will use the timezone chosen by rails, rather than defaulting to the system timezone.